extensions/applicationinsights-osplugin-js/README.md

```diff
@@ -0,0 +1,54 @@
+# Microsoft Application Insights JavaScript SDK - Operating System Plugin
+
+[![GitHub Workflow Status (main)](https://img.shields.io/github/actions/workflow/status/microsoft/ApplicationInsights-JS/ci.yml?branch=main)](https://github.com/microsoft/ApplicationInsights-JS/tree/main)
+[![Build Status](https://dev.azure.com/mseng/AppInsights/_apis/build/status%2FAppInsights%20-%20DevTools%2F1DS%20JavaScript%20SDK%20web%20SKU%20(main%3B%20master)?branchName=main)](https://dev.azure.com/mseng/AppInsights/_build/latest?definitionId=8184&branchName=main)
+[![npm version](https://badge.fury.io/js/%40microsoft%2Fapplicationinsights-osplugin-js.svg)](https://badge.fury.io/js/%40microsoft%2Fapplicationinsights-osplugin-js)
+
+Microsoft Application Insights OS Plugin
+
+## Description
+1DS OS Plugin provides
+## Basic Usage
+## Configuration
+
+### Setup 
+## Build:
+```
+npm install -g grunt-cli
+npm install
+npm run build --silent
+```
+
+## Run unit tests:
+```
+npm run test
+```
+
+## Contributing
```

Comment:
This should include a Quick start guide on how to "use" this plugin and add it to a running system.

While we will probably "add" this by default in a future AISku version, we will still need to document it separately as some people (including 1DS users) won't get it as part of AISku